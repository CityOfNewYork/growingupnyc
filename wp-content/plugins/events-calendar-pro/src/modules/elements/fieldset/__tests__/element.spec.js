import React from 'react';
import renderer from 'react-test-renderer';
import Fieldset from '../element';

describe( 'Fieldset', () => {
	test( 'should match snapshot', () => {
		const component = renderer.create(
			<Fieldset className="test-class">
				<span>Test children</span>
			</Fieldset>
		);
		expect( component.toJSON() ).toMatchSnapshot();
	} );
} );
