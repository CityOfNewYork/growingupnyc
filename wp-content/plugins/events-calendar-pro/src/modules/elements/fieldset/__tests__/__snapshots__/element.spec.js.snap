// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Fieldset should match snapshot 1`] = `
<fieldset
  className="tribe-editor__events-pro__fieldset test-class"
>
  <span>
    Test children
  </span>
</fieldset>
`;
